doc/sphinx/addendum/micromega.rst

```diff
@@ -442,7 +445,7 @@ obtain :math:`-1`. Thus, by Theorem :ref:`Psatz <psatz_thm>`, the goal is valid.
   This is done thanks to the :term:`cone expression`
   :math:`p_2 + p_1 + 3 \times p_0 \equiv -1`.
 
-  .. coqtop:: all
+  .. coqdoc::
```

Comment:
That's what I recommended as a temporary solution until we figure out how the doc should work. I agree this is a brutal change. But this is a (temporary) compromise to avoid the two other alternatives of:

- Making the refman build depend on the stdlib (so dependent on an external repo).
- Moving the parts of the doc that depend on the stdlib to the stdlib repo.

Reply:
>Making the refman build depend on the stdlib (so dependent on an external repo).

Isn't that a better solution?

Reply:
IINM the doc depending on stdlib is very similar to conveniently running coqide in the developer environment with stdlib and rebuilding stdlib after rebuilding Coq.  If that functionality is broken for any length of time, it may prevent or greatly slow down unrelated development work.  Kind of like the slow and rocky original rollout of dune.  Also we should describe any changes in the commands needed to build/run these.